oci/platform.py

```diff
@@ -7,6 +8,56 @@
 import oci.model as om
 
 
+class OperatingSystem(enum.Enum):
+    '''
+    OperatingSystem contains the values for the 'os' property in an oci multiarch image.
+    See https://go.dev/doc/install/source#environment.
+    '''
+    AIX = 'aix'
+    ANDROID = 'android'
+    DARWIN = 'darwin'
+    DRAGONFLY = 'dragonfly'
+    FREEBSD = 'freebsd'
+    ILLUMOS = 'illumos'
+    IOS = 'ios'
+    JS = 'js'
+    LINUX = 'linux'
+    NETBSD = 'netbsd'
+    OPENBSD = 'openbsd'
+    PLAN9 = 'plan9'
+    SOLARIS = 'solaris'
+    WINDOWS = 'windows'
+
+    @classmethod
+    def contains_value(cls, value: str):
```

Comment:
rather overwrite __contains__
as an (more pythonic) alternative, one could resort to leveraging enum.Enum's autogenerated initialiser's semantics

-> `OperatingSystem('does-not-exist')` will raise `ValueError`